src/Tribe/Template.php

```diff
@@ -767,6 +760,36 @@ public function template( $name, $context = [], $echo = true ) {
 		return $html;
 	}
 
+	/**
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: 9be2780b536deea31f69b95bacb00ec4</sub>